CMakeLists.txt

```diff
@@ -41,6 +41,12 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL Linux)
 	list(APPEND sources ${info_file})
 endif()
 
+if(CMAKE_SYSTEM_NAME STREQUAL Emscripten)
+	# This is a hack to workaround the lack of support for shared libraries in the toolchain file shipped with Emscripten
+	# Once this PR (or similar) lands (https://github.com/emscripten-core/emscripten/pull/16281) we can remove this
+	set_property(GLOBAL PROPERTY TARGET_SUPPORTS_SHARED_LIBS TRUE)
+endif()
```

Comment:
I would put this in either `GodotJoltPrelude.cmake` or `GodotJoltSetup.cmake`, which run before and after the `project(...)` call respectively (and which probably should be included with `CMAKE_PROJECT_INCLUDE[_BEFORE]` instead). I'm not sure whether `CMAKE_SYSTEM_NAME` will be set before `project(...)`, so you might want to put it somewhere in `GodotJoltSetup.cmake` instead.